build.sbt

```diff
@@ -20,13 +20,15 @@ scalacOptions ++=
 
 resolvers += Resolver.bintrayRepo("non", "maven") // for kind-projector
 
-addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.9.3")
+addCompilerPlugin("org.spire-math" %% "kind-projector" % "0.9.4")
```

Comment:
There's an 0.9.5 now.  I don't know why it's better than 0.9.4, but I just bumped it on another project.